libs/blocks/global-navigation/global-navigation.js

```diff
@@ -159,20 +214,66 @@ const closeOnClickOutside = (e) => {
   }
 };
 
+const getUniversalNavLocale = (locale) => {
+  const LANGMAP = {
+    cs: ['cz'],
+    da: ['dk'],
+    de: ['at'],
+    en: ['africa', 'au', 'ca', 'ie', 'in', 'mt', 'ng', 'nz', 'sg', 'za'],
+    es: ['ar', 'cl', 'co', 'cr', 'ec', 'gt', 'la', 'mx', 'pe', 'pr'],
+    et: ['ee'],
+    ja: ['jp'],
+    ko: ['kr'],
+    nb: ['no'],
+    pt: ['br'],
+    sl: ['si'],
+    sv: ['se'],
+    uk: ['ua'],
+    zh: ['cn', 'tw'],
+  };
+
+  if (!locale.prefix || locale.prefix === '/') return 'en_US';
+  const prefix = locale.prefix.replace('/', '');
+  if (prefix.includes('_')) {
+    const [lang, country] = prefix.split('_').reverse();
+    return `${lang.toLowerCase()}_${country.toUpperCase()}`;
+  }
+
+  if (prefix === 'uk') return 'en_GB';
+  const customLang = Object.keys(LANGMAP).find((key) => LANGMAP[key].includes(prefix));
+  if (customLang) return `${customLang.toLowerCase()}_${prefix.toUpperCase()}`;
+
+  return `${prefix.toLowerCase()}_${prefix.toUpperCase()}`;
+};
+
+const convertToPascalCase = (str) => str
+  ?.split('-')
+  .map((s) => s.charAt(0).toUpperCase() + s.slice(1))
+  .join(' ');
+
 class Gnav {
   constructor({ content, block } = {}) {
     this.content = content;
     this.block = block;
 
     this.blocks = {
-      profile: {
-        rawElem: this.content.querySelector('.profile'),
-        decoratedElem: toFragment`<div class="feds-profile"></div>`,
-      },
       search: { config: { icon: CONFIG.icons.search } },
       breadcrumbs: { wrapper: '' },
     };
 
+    // TODO: metadata + project config
+    this.universalNavComponents = getMetadata('universal-nav')?.toLowerCase()?.split(',').map((option) => option.trim())
```

Comment:
Should we have a method a separate method for this?

```js 
// Something such as this
const setupUNav = () => {
  this.universalNavComponents = getMetadata('universal-nav')
    ?.toLowerCase()
    ?.split(',')
    .map((option) => option.trim())
    .filter((component) => Object.keys(CONFIG.universalNav.components).includes(component) || component === 'signup');

  this.useUniversalNav = getMetadata('universal-nav')
    ?.toLowerCase() === 'on' || !!this.universalNavComponents?.length;

  if (this.useUniversalNav) {
    this.blocks.universalNav = toFragment`<div class="feds-utilities"></div>`;
    return;
  }

  this.blocks.profile = {
    rawElem: this.content.querySelector('.profile'),
    decoratedElem: toFragment`<div class="feds-profile"></div>`,
  };
};
```

Reply:
This is a good idea, I will explore in a future iteration.